policy-service/pom.xml

```diff
@@ -36,9 +36,16 @@
         <spring-cloud-starter-ribbon.version>1.4.7.RELEASE</spring-cloud-starter-ribbon.version>
```

Comment:
Changes to service level maven pom file will need to be made.  Review by looking at the other files and make the necessary changes.